query/stdlib/influxdata/influxdb/operators.go

```diff
@@ -117,6 +118,31 @@ func (s *ReadRangePhysSpec) TimeBounds(predecessorBounds *plan.Bounds) *plan.Bou
 	}
 }
 
+type Window struct {
+	every int64
+}
+
+type ReadWindowAggregatePhysSpec struct {
+	plan.DefaultCost
+	ReadRangePhysSpec
+
+	Window     Window
+	Aggregates []string
```

Comment:
Should this be `ProcedureKind` just to keep the types the same? `ProcedureKind` is a string so it is functionally the same thing, but it might help provide a clue for what is meant to be put here.